extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
+:host {
```

Comment:
Please move all these to the quiz container, so it works with or without shadow DOM